### DISCUSSION TITLE
Limiting Max Boosts (Game breaking pervention)

### DISCUSSION THREAD

**Discussion by Life-Is-Great** [sunflower-land/sunflower-land#2330] [Ideas]
First of all.
I want to let everyone know that I feel that I got much negativity thrown at me every time I tried to create more equality.
What I would ask myself each time reading a comment would be this::
Is this comment supporting benefits for only small group of people or is it leading to majority of players getting satisfaction from the game.

![image](https://user-images.githubusercontent.com/108883042/224352088-1d43851f-1d22-47d7-839f-29e6417fa42c.png)
### **I strongly suggest that we limit boosts max effect from nfts.**

**Example:**
- Max 3x loot
- Max 30% instant growth
- Max 250% growth speed
etc...

**If decided to give higher boost possibility:**
- It should be time limited (Example: only 6 months)
- Those time limited boosted nfts could come from treasure island and other extra added events

**What will this do?:**
- Give multiple paths for players to go when purchasing nfts
- Bring limit on how much game can break
- More balanced and controllable environment & SFL economy

**Benifits:**
- Activity & loyalty rewarded
- More personal player choices
- More nfts (That do not risk crashing economy because of time limited boosts)
- Joy to more players
- More activity
- Safe plug for economy
- More burn
and more...

----------------------------------------------------------------------

SuperSynapse is suggesting following cap.
He does have great ideas and I do like it as long as it is in sync with SFL economy being healthy 

1x (starting player)
2x (beginning player)
4x (advanced player)
6x (end game player)

**Comment by offerT** [comment]
The bonus can be related to grade, seasons and sfl economy
Maybe we can increase crab nft by 10% in summer and 5% in other seasons

**Reply by Life-Is-Great** [reply]
I like this to.
But in this case. I would love to make it cost to reactivate next season.

Idea is to not overwhelm economy and undervalue SFL token by all permanent boosts created overtime.

**Comment by ghost** [comment]
A very strange idea, what's the point then in selling and buying SFT if you have a limit?  If you give SFT boosts only for a certain time, it will greatly reduce its cost and the value of the boost.

**Reply by Life-Is-Great** [reply]
### You would have more personal choices and multiple paths of achieving what you like.

**Today:**
* Every nft that comes out is stocking on top of existing boosts.
* This is making it very hard to introduce new nfts for specific group of players.
* Those extracting most resources will extract even more and if they quit it will do more harm to SFL as a whole

----------------------------------
**Today**
* Already owned boosts = 300% more crops

* Next NFT released (200 stock)
** This nft is giving 100% more resources

* Next NFT released (300 stock)
** This nft is giving 50% more resources

etc etc...

**What this is leading to is that only 1 group of people get all the nfts and extract value.**

-----------------------------------

**Future:**
* Already owned boosts = 100% more crops

* Next NFT released (200 stock)
** This nft is giving 100% more resources

* Next NFT released (300 stock)
** This nft is giving 50% more resources

etc etc...

**What this is leading to is that more players are satisfied and can participate in the game.**

------------------------------------


And if developers want to reward activity...
They can give **time limited boosts** to active/loyal/whale players
This leads to lazy people not draining SFL just because they where active once for few months 😂 




Keep in mind that limits can be raised in future if fits SFL economy

**Reply by ghost** [reply]
Now that seasons have been added to the game, SFT will be taken not only by "Whales", but also by new players. 

What to do with SFTS that cost 500+$? how will they work?

**Reply by ghost** [reply]
Why a player who invested less money.
Should have the same boosters as me?

**Reply by Life-Is-Great** [reply]
>What to do with SFTS that cost 500+$? how will they work?

They will give boost that they do

This does not mean that it will lose value because it isn't a must to reach max boost even when you own all nfts & having to buy 1 nft instead of 10 nfts will sure have its value


*_Keep in mind that I'm only giving you a better view of what you are wondering about in a situation that you are suggesting_

**Reply by Life-Is-Great** [reply]
>Why a player who invested less money.
Should have the same boosters as me?

no one said that he/she should

**Comment by NilzGone** [comment]
I'm a bit confused 

What will this do?:
_Give multiple paths for players to go when purchasing nfts_   - Or limit them to one path as they can't make use of anymore NFTs
_Bring limit on how much game can break_ - Maybe but this should be judged by the number of each new NFT, it shouldn't break the game because someone might use them all
_More balanced and controllable environment & SFL economy_ Maybe but in a way removes part of game play as you have to work out your own balance

Benifits:
_Activity & loyalty rewarded_ - But they're limited to a number of NFTS? What if a new one comes out that they want to use as well, this is more for newer players
_More personal player choices_ - By being limited by the number of paths you can go down?
_More nfts (That do not risk crashing economy because of time limited boosts)_ - That less people will be able to make use of, thus lowering the market for them
_Joy to more players_ - Not being able to use a new item coming out? and being limited to the path you're on unless you sell?
_More activity_ - Less people could mint new items = less activity
_Safe plug for economy_ - Also lowers market activity on NFTS
_More burn_ - Less burn as less people rushing to get new NFTS
and more...

Please explain more

**Reply by Life-Is-Great** [reply]
I replied in more detail here:
https://github.com/sunflower-land/sunflower-land/discussions/2330#discussioncomment-5276896

**Comment by LandLadyFarmer** [comment]
You can do a lot with boost and area boost is coming but have any time limiting boost is nothing i would like to have and just opens up for bad things to happen and secondhand market would be crazy when all people want to have boost time all the time.....
People payed high prices for theirs boost so then nerf it any way that could literally kill the price of it is just and absolute nothing we even should think of...
we can not predict what longtime effect it could have if we start with it, that opens up for devs to kill any nft at any time..
Any released item should NOT be altered.. That would be like you bought a car and the dealer comes and change your motor to something worse... 
we have halving already that balance away high boost item so we don't need anything else..
Have seen it being talked about the wings and there is no problem with them at all and if there is 10-20 people that race to an mint faster than others so what yours nft makes you go faster than others so should we nerf them to?, Just be happy for wing owners and let the enjoy them it's part of the game and those play the game as we do......

So i say no to any time limiting boost in any form i see only problem with it and no fun specially in an idle game like this and we shouldn't talk about secondhand market,, I can't see that would bring any more fun to play i get the opposite feeling...
I chose this game for the idle play style and devs idea and path that i still like and consider should keep going, there are some small problems here and there but for an rolling release i think it's pretty darn good and will get better as we go...

People just gets more and more unusual ideas, i don't no whats happened!! Did it change to much for OG players with land expansion or what happened??? This concerns a small group of OG players that went crazy and now comes up with one after another thing they try to change and complain about, discord went totally negative and no fun anymore...

**Reply by Life-Is-Great** [reply]
I do not se this in reality what you are saying sorry.
And also. It would be much nicer for me to reflect if you could organize what you are saying in to parts that I can comment on.

>pens up for bad things to happen 

What bad things can happen specifically?

>secondhand market would be crazy when all people want to have boost time all the time

Not really (Only if core team decide to release enough nfts for all players to obtain those boosts)

>People payed high prices for theirs boost so then nerf it any way that could literally kill the price

No one would nerf any nft. There would be a max limit to all boosts combined together.

>we can not predict what longtime effect it could have if we start with it, that opens up for devs to kill any nft at any time

With this devs would have better control over where SFL economy is heading and therefore SFL will be able to exist in 10 years from now on.

>Any released item should NOT be altered

Agree. What I'm suggesting isn't doing that

>Have seen it being talked about the wings

Yes. As for other nfts that created totally unbalanced market for every other player not owning them


I would love to say that every player is valuable. Not only few.
